### PR TITLE
Removes size and status from harvest source model

### PR DESCRIPTION
**Pull request by btylerburton**
# Pull Request

Related to https://github.com/GSA/data.gov/issues/4784

## About

Removes size and status from harvest source model in favor of passing in same val for memory and disk to all tasks, with eventual focus on shaving memory requirements off after proper profiling

## PR TASKS

- [X] The actual code changes.
- [X] Tests written and passed.
- [ ] Any changes to docs?

### CODE REVIEW

File: app/scripts/load_manager.py [hunk 1]
```diff
@@ -10,19 +10,13 @@
 CF_SERVICE_USER = os.getenv("CF_SERVICE_USER")
 CF_SERVICE_AUTH = os.getenv("CF_SERVICE_AUTH")
 HARVEST_RUNNER_APP_GUID = os.getenv("HARVEST_RUNNER_APP_GUID")
-CF_INSTANCE_INDEX = os.getenv("CF_INSTANCE_INDEX")
```
Comment (rshewitt) [inline app/scripts/load_manager.py:13]: why is this removed?

File: harvester/lib/cf_handler.py [hunk 2]
```diff
@@ -14,10 +15,10 @@ def setup(self):
         self.client.init_with_user_credentials(self.user, self.password)
         self.task_mgr = TaskManager(self.url, self.client)
 
-    def start_task(self, app_guuid, command, task_id, memory_in_mb=512, disk_in_mb=512):
-        return self.task_mgr.create(
-            app_guuid, command, task_id, memory_in_mb, disk_in_mb
-        )
+    def start_task(self, app_guuid, command, task_id):
+        TASK_MEMORY = os.getenv("HARVEST_RUNNER_TASK_MEM", "4096")
```
Comment (btylerburton) [inline harvester/lib/cf_handler.py:19]: I can't figure out how to mock these if I put them at the head of the file. I know that gets calculated at import time vs execution time, and [I could use `pytest-env` if I wanted them to be a single, static value](https://github.com/pytest-dev/pytest/discussions/10027), but does anyone on @GSA/data-gov-dev-team know how to monkeypatch them for a single test?

File: app/scripts/load_manager.py (hunk 1 above)
Comment (btylerburton) [reply app/scripts/load_manager.py:13]: it's imported manually here: https://github.com/GSA/datagov-harvester/pull/96/files#diff-8304ae22782c8fdf080c8374d1a6431e2ca7b7a9e4050dd71540fd7e67532072R98 so this is redundant

File: harvester/lib/cf_handler.py (hunk 2 above)
Comment (rshewitt) [reply harvester/lib/cf_handler.py:19]: does memory and disk space need to derive from the env? could they be default args similar to what you had previously?

Comment (btylerburton) [reply harvester/lib/cf_handler.py:19]: i'd like them to be configurable, at least while we are trying to nail down what the correct values should be.

**Comment by btylerburton** [comment]
Any further changes or is this good to merge @rshewitt?

**Review by rshewitt — APPROVED** [review]
